frontend/src/components/forms/FormField.module.scss

```
@@ -25,4 +25,6 @@
   path {
     fill: $COLOR_RASPBERRY;
   }
+  overflow: hidden;
+  text-overflow: ellipsis;
```

```suggestion
  & > span {
    overflow: hidden;
    text-overflow: ellipsis;
  }
  & > svg {
    min-width: max-content;
  }
```

The overflow styles should work nicer on the span. With the current styles it's the span that overflows the div, not the text content, so the text overflow doesn't take effect.

Also noticed that the icon shrinks, it would be nice to keep constant size.